patient-search/ui/src/pages/patientProfile/Demographics.tsx

```diff
@@ -0,0 +1,129 @@
+import { useEffect, useState } from 'react';
+import { TableComponent } from '../../components/Table/Table';
+import { Button, Icon } from '@trussworks/react-uswds';
+
+export const Demographics = () => {
+    const [tableBody, setTableBody] = useState<any>([]);
+    const [nameTableBody, setNameTableBody] = useState<any>([]);
+    const [currentPage, setCurrentPage] = useState<number>(1);
+
+    useEffect(() => {
+        const tempArr = [];
+        for (let i = 0; i < 3; i++) {
+            tempArr.push({
```

Comment:
This hard coded data is going to be removed anyways. 
But in future may be we can think about having a fixtrure with sample.json while the api is not ready. We can use the fixture sample. And we can leave the fixture for local development purposes.

Reply:
Yes also agree with this style of implementation for usage of dummy data. will integrate onwards